great_expectations/rule_based_profiler/domain_builder/column_domain_builder.py

```diff
@@ -237,38 +233,16 @@ def get_effective_column_names(
                     message=f'Error: The column "{column_name}" in BatchData does not exist.'
                 )
 
-        # include_column_name_suffixes column_name_suffixes from "rule state" (i.e., variables and parameters); from instance variable otherwise.
-        include_column_name_suffixes: Optional[
-            Union[str, Iterable, List[str]]
-        ] = get_parameter_value_and_validate_return_type(
-            domain=None,
-            parameter_reference=self.include_column_name_suffixes,
-            expected_return_type=None,
-            variables=variables,
-            parameters=None,
-        )
-
-        # exclude_column_name_suffixes column_name_suffixes from "rule state" (i.e., variables and parameters); from instance variable otherwise.
-        exclude_column_name_suffixes: Optional[
-            Union[str, Iterable, List[str]]
-        ] = get_parameter_value_and_validate_return_type(
-            domain=None,
-            parameter_reference=self.exclude_column_name_suffixes,
-            expected_return_type=None,
-            variables=variables,
-            parameters=None,
+        include_column_name_suffixes: Union[str, Iterable, List[str]] = cast(
```

Comment:
can we please leave this type as a list of string? Every time we go with a multitude of types, it creates a multitude of if statements later on

Reply:
@donaldheppner It can take on multiple time (I do not think this changed) -- happy to discuss in case I am not seeing the exact line(s) you have in mind.  Thanks.

Reply:
@donaldheppner This was done.